app/package.json

```diff
@@ -52,9 +53,11 @@
     "luxon": "^3.0.1",
     "next": "^12.1.6",
     "openid-client": "^5.1.6",
+    "patch-package": "^6.5.0",
     "path-to-regexp": "^6.2.1",
     "pluralize": "^7.0.0",
     "postgraphile": "^4.12.9",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
Yarn runs the postinstall script after `yarn`, `yarn install` and `yarn add <package>` but not after `yarn remove <package>`. This postinstall hook will run even after a `yarn remove <package>`.